File: clients/python/tests/client_test.py
import os
from time import sleep
from selenium import webdriver
from selenium.webdriver.chrome.service import Service
from session_client import init_session

def test():
    driver = webdriver.Chrome('clients/java/chromedriver.exe')
    init_session(driver, ['github'])
    driver.get("https://github.com/")
    sleep(5)
    driver.quit()

test()

File: clients/javascript/tests/client-test.js
import { Builder, By, Key } from 'selenium-webdriver';
import chrome from 'selenium-webdriver/chrome.js';
import * as seleniumSessionClient from '../src/selenium-session-client.js';

let service = new chrome.ServiceBuilder('../java/chromedriver').build();
chrome.setDefaultService(service);
let driver = new Builder().forBrowser('chrome').build();

await seleniumSessionClient.initSession(driver, ['github']);
await driver.get('https://github.com');
setTimeout(async () => {
    await driver.quit();
}, 5000);